hpOneView/oneview_client.py

```diff
@@ -416,40 +416,48 @@ def id_pools_vsn_ranges(self):
         Gets the IdPoolsVsnRanges API client.
```

Comment:
I suggest change the descriptions of the following methods to:
* id_pools_vsn_ranges = Gets the IdPoolsRanges API Client for VSN Ranges
* id_pools_vmac_ranges = Gets the IdPoolsRanges API Client for VMAC Ranges
* id_pools_vwwn_ranges= Gets the IdPoolsRanges API Client for VWWN Ranges